Interop/Indexer/src/main/kotlin/org/jetbrains/kotlin/native/interop/indexer/Indexer.kt

```diff
@@ -522,7 +522,7 @@ internal class NativeIndexImpl(val library: NativeLibrary, val verbose: Boolean
             val size = clang_Type_getSizeOf(type)
             val elemSize = clang_Type_getSizeOf(elementCXType)
             val elementCount = clang_getNumElements(type)
-            assert(elemSize * elementCount == size)
+            assert(size >= elemSize * elementCount && size % elemSize == 0L)
```

Comment:
Could you provide an example violating previous version of assertion?

Reply:
simd_float3 and similar has internal padding and sizeof == 16